### PR TITLE
Update The Team with new user groups

### PR DESCRIPTION
**Pull request by cl8n**
and some other minor fixes. forgot to do this after adding Project Loved Team

---

### CODE REVIEW

**Comment by cl8n** [comment]
sure but different PR, here I just want to get the missed update in

also why is this blocked

File: wiki/People/en.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ The following are user groups which are made up of the osu! community members th
 
 - [Beatmap Nominators](/wiki/People/Beatmap_Nominators)
 - [Community Contributors](/wiki/People/Community_Contributors)
+<!-- This should be its own page under People -->
```
Comment (TPGPL) [inline wiki/People/en.md]: maybe at this in the same line as Developers?

Comment (cl8n) [reply wiki/People/en.md]: does it matter it's just a comment 🤔

Comment (TPGPL) [reply wiki/People/en.md]: thats why making a seperate line just for it seems unnecessary 🤷‍♂

**Pull request merged**